README.rst

```diff
@@ -58,6 +59,8 @@ Next, install LingoX and configure it:
   language for a specific site: go to Sites
   Configuration ``/admin/site_configuration/siteconfiguration/``, add a ``LANGUAGE_CODE`` key with the desired
   site-specific value to the site's configuration JSON.
+  To enable or disable the feature on a specific site, define the variable ``ENABLE_LINGOX`` in the site's
```

Comment:
@OmarIthawi We should clarify that the variable goes under the `FEATURES` variable dict.

Reply:
Not really @melvinsoft, this is for the `Site` configuration in the admin panel. `FEATURES` only exist in the `aws.py` configurations.

What do you think?

Reply:
Sorry, you're absolutely right @OmarIthawi. Ship it!